hadoop-common-project/hadoop-common/src/site/markdown/filesystem/filesystem.md

```diff
@@ -432,7 +432,7 @@ of data which must be collected in a single RPC call.
 
 #### Preconditions
 
-    exists(FS, path) else raise FileNotFoundException
+    if not exists(FS, path) : raise FileNotFoundException
```

Comment:
I was using the concept exists() holds or raise FNFE, but if this suits then I'm not going to argue about details. The goal of this spec is for other people and tests, not mathematical purism.

Reply:
Either form is understandable imo, just thought the suggested form is more in line with the remainder of the docs which uses pythonesque code.